### ISSUE TITLE
electrum-dash-server repo missing issues section

### ISSUE THREAD

**Issue by guruvan** [dashpay/electrum-dash#19]
seems like an ok place to put this - maybe someone can set that up :)

**Comment by guruvan** [comment]
@kefkius - I wanted to report an issue with IRC connection, so for now I'll stash it here. 

I'm getting connection reset by peer on all my connections. I thought I saw this working, but it's not now

I'm running from dashpay/electrum-dash-server master with an image built to accommodate a few script issues. Can you take a look and see if you can reproduce and if so, if you can fix? 

I've tested with a couple different irc_nick settings (to make sure freenode's not kicking me out because multiple users with same nick

**Comment by Kefkius** [comment]
Where is this message being logged? In the terminal you're running electrum-dash-server from?

**Comment by guruvan** [comment]
yeah - the electrum server runs in the foreground in the container, so all the stdout goes straight to the docker logs. Thanks :))  - I'll also double-check an older container revision - I really think there's one I had with working irc connections, but tbh had not verified on irc

**Comment by guruvan** [comment]
I'll post a stack trace here in a little bit as well

**Comment by guruvan** [comment]
lots of incorrect checksum errors. doesn't seem to be iptables/docker related, have tried with `--net host` in docker and see the same results - will try with another host OS later today

  File "/app/src/ircthread.py", line 138, in run
    client.process_forever()
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 278, in process_forever
    self.process_once(timeout)
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 259, in process_once
    self.process_data(i)
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 216, in process_data
    c.process_data()
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 573, in process_data
    self.disconnect("Connection reset by peer")
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 779, in disconnect
    self._handle_event(Event("disconnect", self.server, "", [message]))
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 673, in _handle_event
    self.reactor._handle_event(self, event)
  File "build/bdist.linux-x86_64/egg/irc/client.py", line 398, in _handle_event
    result = handler.callback(connection, event)
  File "/app/src/ircthread.py", line 90, in on_disconnect
    raise BaseException("disconnected")


  ```18:26:30.758195 IP (tos 0x0, ttl 51, id 42515, offset 0, flags [DF], proto TCP (6), length 109)
    82.96.64.4.6667 > 172.17.0.6.42012: Flags [P.], cksum 0xc88f (correct), seq 4021979088:4021979145, ack 3996398577, win 453, options [nop,nop,TS val 2708350673 ecr 85131803], length 57
18:26:30.758215 IP (tos 0x0, ttl 64, id 13247, offset 0, flags [DF], proto TCP (6), length 52)
    172.17.0.6.42012 > 82.96.64.4.6667: Flags [.], cksum 0x3ea2 (incorrect -> 0xf9ab), seq 1, ack 57, win 229, options [nop,nop,TS val 85137484 ecr 2708350673], length 0```
